package.json

```diff
@@ -33,7 +33,7 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "admin_extract_messages": "ngx-translate-extract -k -c -s -i ./projects/admin/src -o ./projects/admin/src/app/translate/i18n/en_US.json --fi '  '",
-    "admin_update_catalog": "ngx-translate-extract -k -c -s -i ./projects/admin/src -o ./projects/admin/src/app/translate/i18n/{de,en,fr,it}.json --fi '  '",
+    "admin_update_catalog": "ngx-translate-extract -k -c -s -i ./projects/admin/src -o ./projects/admin/src/app/translate/i18n/{de,en,es,fr,it}.json --fi '  '",
```

Comment:
You could mention in commit message that a new language is added.

Reply:
done